src/plugins/physics/EC_RigidBody_Ammo.js

```diff
@@ -0,0 +1,753 @@
+
+define(["core/framework/TundraSDK",
+        "lib/ammo",
+        "lib/three",
+        "core/scene/Scene",
+        "core/scene/Attribute",
+        "core/scene/AttributeChange",
+        "core/physics/CollisionInfo",
+        "entity-components/EC_RigidBody"
+    ], function(TundraSDK, Ammo, THREE, Scene, Attribute, AttributeChange, CollisionInfo, EC_RigidBody) {
+
+/**
+    @class EC_RigidBody_Ammo
+    @extends EC_RigidBody
+    @constructor
+*/
+var EC_RigidBody_Ammo = EC_RigidBody.$extend(
+{
+    __init__ : function(id, typeId, typeName, name)
+    {
+        this.$super(id, typeId, typeName, name);
+        
+        // TODO! use btMotionState subclass if possible
```

Comment:
I've been using `@todo` and `@note` for stuff like this. Just easier to grep the repo and find all instances if we try to use the sane naming.

Note that if you use @todo/note inside `/** */` doc string it will show up in the public docs, which might not always be what you want. (I've been bitten by this :)